src/strategies/diadata/index.ts

```diff
@@ -0,0 +1,61 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'diadata';
+export const version = '0.1.0';
+
+
+const abi = [
+    {
+      constant: true,
+      inputs: [
+        {
+          internalType: 'address',
+          name: 'account',
+          type: 'address'
+        }
+      ],
+      name: 'balanceOf',
+      outputs: [
+        {
+          internalType: 'uint256',
+          name: '',
+          type: 'uint256'
+        }
+      ],
+      payable: false,
+      stateMutability: 'view',
+      type: 'function'
+    }
+  ];
+  
+  export async function strategy(
+    network,
+    provider,
+    addresses,
+    options,
+    snapshot
+  ) {
+    const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+    const response = await multicall(
+      network,
+      provider,
+      abi,
+      addresses.map((address: any) => [options.address, 'balanceOf', [address]]),
```

Comment:
Can you import the strategy `erc20-balance-of` here like this: https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of-coeff/index.ts#L13
This help reduce duplicate code. You could also copy this part https://github.com/snapshot-labs/snapshot.js/blob/master/src/strategies/erc20-balance-of-coeff/index.ts#L20-L25 and just change this line
```
address[1] * options.coeff
```
to this
```
address[1] > options.minTokenBalance ? 1 : 0
```
Also can you change the name of the strategy to be more generic, i recommend 'erc20-balance-of-minimum'

Reply:
Thanks, much cleaner now - done!